native_store.go

```diff
@@ -29,10 +31,18 @@ const (
 	emptyUsername           = "<token>"
 )
 
+// dockerCredentials mimics how docker credential helper binaries store
+// credential information.
+type dockerCredentials struct {
+	ServerURL string
+	Username  string
+	Secret    string
```

Comment:
It is better to explicitly declare the json field name. For example:
```go
ServerURL string `json:"ServerURL"`
```

Reply:
Added the json field name.